sdk/src/main/java/com/kin/ecosystem/onboarding/view/OnboardingFragment.kt

```diff
@@ -76,7 +76,7 @@ class OnboardingFragment: Fragment(), IOnboardingView {
         with(loadingAVD) {
             visibility = View.VISIBLE
             animate().alpha(AnimConsts.Value.ALPHA_1).setDuration(AnimConsts.Duration.FADE_ANIM).start()
-            (drawable as AnimatedVectorDrawable).start()
+            (drawable as AnimatedVectorDrawableCompat).start()
```

Comment:
Just for my knowledge, why did you went to Compat instead of the previous one?

Reply:
to support below API 21